flink-end-to-end-tests/flink-end-to-end-tests-common-kafka/pom.xml

```diff
@@ -224,6 +223,15 @@ under the License.
 							<type>jar</type>
 							<outputDirectory>${project.build.directory}/dependencies</outputDirectory>
 						</artifactItem>
+						<artifactItem>
```

Comment:
We can remove this snippet if we move `ListSource` (`FromElementsSource`) to `flink-streaming-java`